examples/render-helm-chart-remote-values-file/README.md

```diff
@@ -0,0 +1,46 @@
+# render-helm-chart: Remote Values File
+
+### Overview
+
+This example demonstrates how to imperatively invoke the `render-helm-chart`
+function with a remote values file.
+
+### Function invocation
+
+Run the following command to fetch the example package:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/render-helm-chart-remote-values-file
+```
+
+```shell
+$ cd render-helm-chart-remote-values-file
```

Comment:
You don't have to do this if you run `kpt fn eval render-helm-chart-remote-values-file`.

Reply:
changed!